Added Danish language messages

jowr opened the pull request.

jowr committed:
Added Danish language messages

bovender commented:
Thanks for submitting this. It made me realize that the messages contain the name of my XL Toolbox add-in for Excel, even though the script is meant to be rather generic.

Have you seen the successor to this script, https://github.com/bovender/VstoAddinInstaller? It's much better than ExcelAddinInstaller (but does not work with VBA add-ins).

By the way, the apostrophe in "Daniel's XL Toolbox" was deliberate even in the German translation, because it's really a proper name. (Likewise, it's McDonald's, not McDonalds even in the German 'restaurants'.) Would you mind changing this back?

jowr committed:
Fixing the apostrophe 

The apostrophe is apparently part of a name and should thus be included in German and Danish, covers https://github.com/bovender/ExcelAddinInstaller/pull/2

jowr commented:
OK, thanks for the explanation, https://github.com/CoolProp/ExcelAddinInstaller/commit/a434971d1eac28bb2b4009bc94b340cbb1f1ad59 should fix this.
